packages/openapi-framework/index.ts

```diff
@@ -187,6 +187,7 @@ export default class OpenAPIFramework implements IOpenAPIFramework {
         : null;
 
     const paths = [].concat(this.paths);
+    this.logger.debug(`${this.loggingPrefix}paths=`, paths);
```

Comment:
with node `console.debug`, is there a way to disable the output in the logs?  the [docs](https://nodejs.org/api/console.html#console_console_debug_data_args) suggest that `console.debug` is merely an alias for `console.log`